config.go

```diff
@@ -40,6 +40,9 @@ type Config struct {
 
 	IPBlocklist iplist.Ranger
 	DisableIPv6 bool `long:"disable-ipv6"`
+	LimitSendPieceRate bool `long:"limit-send-piece-rate"`
+	// how many kB can be send every second
+	SendPieceRate uint32 `long:"max-kbyte-can-send-every-second"`
```

Comment:
A simpler approach would be to have just the `SendPieceRate` variable, and have the value `0` to represent no limiting.

Also I'm confused why it's unsigned, negative values have no meaning here.